src/Parser.php

```diff
@@ -660,6 +636,64 @@ private function parseTableRows(): array
         return $table;
     }
 
+    /**
+     * @param TToken $keywordToken
+     *
+     * @return array{title:string|null, description:string|null}
+     */
+    private function parseTitleAndDescription(array $keywordToken): array
+    {
+        $originalTitle = trim($keywordToken['value'] ?? '');
+        $textLines = [];
+
+        while (in_array($predicted = $this->predictTokenType(), ['Newline', 'Text', 'Comment'])) {
+            $token = $this->expectTokenType($predicted);
+
+            if ($token['type'] === 'Comment') {
+                continue;
+            }
+
+            $text = match ($token['type']) {
+                'Newline' => '',
+                'Text' => $token['value'],
+                default => throw new LogicException('Unexpected token type: ' . $token['type']),
+            };
+
+            if ($this->compatibilityMode->shouldRemoveDescriptionPadding()) {
+                $text = preg_replace('/^\s{0,' . ($keywordToken['indent'] + 2) . '}|\s*$/', '', $text);
```

Comment:
This is the regex that the individual methods were originally using to un-indent / trim text lines when parsing their nodes.